reporter/src/funTest/kotlin/reporters/NoticeSummaryReporterTest.kt

```diff
@@ -52,90 +52,62 @@ private fun generateReport(
 class NoticeSummaryReporterTest : WordSpec({
     "NoticeReporter" should {
         "generate the correct license notes" {
-            val expectedText = File("src/funTest/assets/NPM-is-windows-1.0.2-expected-NOTICE").readText()
-            val ortResult = readOrtResult("src/funTest/assets/NPM-is-windows-1.0.2-scan-result.json")
-
-            val report = generateReport(ortResult)
-
-            report shouldBe expectedText
-        }
-
-        "contain all licenses without excludes" {
-            val expectedText = File("src/funTest/assets/npm-test-without-exclude-expected-NOTICE").readText()
-            val ortResult = readOrtResult("src/funTest/assets/npm-test-without-exclude-scan-results.yml")
-
-            val report = generateReport(ortResult)
-
-            report shouldBe expectedText
-        }
+            val expectedText = File("src/funTest/assets/notice-summary-reporter-expected-results").readText()
 
-        "not contain licenses of excluded packages" {
-            val expectedText = File("src/funTest/assets/npm-test-with-exclude-expected-NOTICE").readText()
-            val ortResult = readOrtResult("src/funTest/assets/npm-test-with-exclude-scan-results.yml")
-
-            val report = generateReport(ortResult)
+            val report = generateReport(ORT_RESULT)
 
             report shouldBe expectedText
         }
 
         "evaluate the provided pre-processing script" {
-            val expectedText = File("src/funTest/assets/pre-processed-expected-NOTICE").readText()
-            val ortResult = readOrtResult("src/funTest/assets/NPM-is-windows-1.0.2-scan-result.json")
+            val expectedText =
+                File("src/funTest/assets/notice-summary-reporter-pre-processed-expected-results").readText()
 
+            // Filter out all MIT findings.
```

Comment:
Why?

Reply:
So that the script affects the generated notices, I have added that to the comment.

Reply:
Actually I found your "affects the licenses contained in the generated notices" here better than what you wrote in the code, because "affects" makes clearer that the data *is* actually changed. If you just say "not contained in the output" it could have been that the licenses were not contained in the input to begin with. But that's just FYI, I've approved anyway.